manifests/globals.pp

```diff
@@ -28,64 +28,23 @@
-#   If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param mongod_service_manage
-#   This setting can be used to override the default management of the mongod service.
-#   By default the module will manage the mongod process.
-# @param service_enable
-#   This setting can be used to specify if the service should be enable at boot
-#
-# @param service_ensure
-#   This setting can be used to specify if the service should be running
-#
-# @param service_name
-#   This setting can be used to override the default MongoDB service name.
-#   If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param service_provider
-#   This setting can be used to override the default MongoDB service provider.
-#   If not specified, the module will use whatever service provider is the default for your OS distro.
-#
-# @param service_status
-#   This setting can be used to override the default status check command for your MongoDB service.
-#    If not specified, the module will use whatever service name is the default for your OS distro.
-#
-# @param user
-#   This setting can be used to override the default MongoDB user and owner of the service and related files in the file system.
-#   If not specified, the module will use the default for your OS distro.
-#
-# @param group
-#   This setting can be used to override the default MongoDB user group to be used for related files in the file system.
-#   If not specified, the module will use the default for your OS distro.
-#
-# @param ipv6
-#   This setting is used to configure MongoDB to turn on ipv6 support.
-#   If not specified and ipv6 address is passed to MongoDB bind_ip it will just fail.
-#
-# @param bind_ip
-#   This setting can be used to configure MonogDB process to bind to and listen for connections from applications on this address.
-#   If not specified, the module will use the default for your OS distro.
-#   Note: This value should be passed as an array.
-#
 # @param version
-#   The version of MonogDB to install/manage. This is needed when managing repositories.
-#   If not specified, the module will use the default for your OS distro.
+#   The version of MonogDB to install/manage.
+#   If not specified, the module will ensure packages with `present`.
+#
+# @param manage_package_repo
+#   Whether to manage MongoDB software repository.
 #
 # @param repo_version
 #   The version of the package repo.
 #
-# @param manage_package_repo
-#   Whether to use the MongoDB software repository or the OS packages (True) or a Custom repo (False)
+# @param use_enterprise_repo
+#   When manage_package_repo is set to true, this setting indicates if it will use the Community Edition
+#   (false, the default) or the Enterprise one (true).
 #
-# @param manage_package
-#   wgether this module willm manage the mongoDB server package
```

Comment:
I removed this because the param because I couldn't make eny sense out of it.
It only controlled package naming.